andela/caret-bn-backend
# #169107967 - Fixing Heroku database

#### What does this PR do?
Fixing Heroku database

#### Description of Task to be completed?
- Edit src/database/config.js to use
- Edit swaggerurl to fetch from environment variables
- Install `sequelize-cli` in the dependencies

#### How should this be manually tested?
- Clone this repo `git clone https://github.com/andela/caret-bn-backend.git`
- Navigate to root folder and run `npm i` to install all dependencies
- The `src/database/config.js` should look like this 
```
require('dotenv').config();

module.exports = {
  development: {
    use_env_variable: 'DATABASE_URL',
    dialect: 'postgres',
    logging: false,
  },
  test: {
    use_env_variable: 'TEST_DB',
    dialect: 'postgres',
    logging: false,
  },
  production: {
    use_env_variable: 'DATABASE_URL',
    dialect: 'postgres',
    logging: false,
  }
};
```
- Explore `.env.example` and mirror it in your `.env` file
- Run `npm test`, all tests should be passing `green`.

#### Any background context you want to provide?
N/A

#### What are the relevant pivotal tracker stories?
[#169107967](https://www.pivotaltracker.com/story/show/169107967)

#### Screenshots (if appropriate)
N/A

#### Questions:
N/A
